doc/loader.rst

```diff
@@ -5,128 +5,539 @@ MIG LOADER
+password to use. This allows for different credentials per endpoint to connect
+to the relay, and provides an ability to isolate an agent from connecting if
+needed which would not be possible with a shared credential.
+
+Instead of putting the credentials in the configuration file, your relay configuration
+line should look something like this.
+
+::
+
+    relay = "amqps://<<AMQPCRED>>@my.mig.relay.url:5671/mig"
+
+<<AMQPCRED>> will be replaced on the endpoint with whatever credentials that
+endpoint should be using to connect to the relay.
+
+**Note:** If you want a built-in configuration, you will not be able to do per-agent
+relay credentials (all agents will connect with the same AMQP username and password).
+
+To create a manifest, create a directory we will use to place the files we want
+to be in the manifest. Copy the components into the directory you want to be part
+of the manifest. The components must have specific file names representing their
+function. The directory name can be anything.
+
+============= =======================================
+Filename      Component
+------------- ---------------------------------------
+mig-agent     The MIG agent binary you want to deploy
+mig-loader    The MIG loader binary you want to deploy
+configuration Agent configuration file
+cacert        CA certificate
+agentcert     Agent certificate to connect to relay
+agentkey      Agent key to connect to relay
+============= =======================================
+
+When creating a manifest, you will likely end up with something like this.
+
+::
+
+    $ cd mig-manifest-int-linux
+    $ ls
+    agentcert  agentkey  cacert  configuration  mig-agent  mig-loader
+    $
+
+To finish creating our manifest we will use, tar/compress the directory into
+the manifest file we will upload to the API.
+
+::
+
+    $ tar -czvf mig-manifest-linux.tar.gz mig-manifest-int-linux
+    mig-manifest-int-linux/
+    mig-manifest-int-linux/mig-loader
+    mig-manifest-int-linux/configuration
+    mig-manifest-int-linux/mig-agent
+    mig-manifest-int-linux/agentcert
+    mig-manifest-int-linux/cacert
+    mig-manifest-int-linux/agentkey
+    $
+
+Creating a new manifest in the API
+----------------------------------
+Next we need to send our new manifest to the API, so it is available to be
+fetched by loader instances we are running. This is accomplished using
+mig-console.
+
+A MIG administrator account is required for this. To make an account an
+administrator account, the ``isadmin`` column for the investigator in the
+``investigators`` table should be set to ``true``.
+
+The ``create manifest`` command is used to create the new manifest.
+
+::
+
+    mig> create manifest
+    Entering manifest creation mode.
+    Please provide the name of the new manifest
+    name> a new manifest
+    Name: 'a new manifest'
+    Please provide loader targeting string for manifest.
+    target> env#>>'{os}'='linux'
+    Target: 'env#>>'{os}'='linux''
```

Comment:
All environments have the `os` flag set so you could use `env->>'os'='linux'` instead.